superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -166,14 +210,148 @@ export default class ResultSet extends React.PureComponent<
     }
   }
 
+  handleOverwriteDatasetOption(data, option) {
```

Comment:
what's the `data` arg?

Reply:
its a method from antd autocomplete that just returns the value, but I want the entire object so i can reference the datasetId